scripts/tokenize_data.sh

```diff
@@ -1,3 +1,4 @@
 python3 \
-    ../../tokenize_data.py \
-    ../../configs/user_configs/<YOUR_CONFIG_HERE>.yaml
+    ../../src/tokenize_data.py \
+    ../../configs/user_configs/<YOUR_CONFIG_HERE>.yaml \
+    <YOUR_SPLIT_HERE> # train, validation, or test. You can start a job for each split.
```

Comment:
Again, the question remains why we don't have this inside the config file itself. I just want a reason, that is all.

Reply:
Parallelization. Rather than having one job that does train, test, and validation, you can start 3 jobs on separate nodes that do them all at once. For C4, it takes just over 24 hours to tokenize all of the data. You can only have a job go for 24 hours, so it makes sense to break up the data a bit into different jobs.

This isn't a parameter we would want to put in the config, because it's not a persistent parameter across multiple scripts.